### DISCUSSION TITLE
I get the error message 'cp949'

### DISCUSSION THREAD

**Discussion by MirrorPayne** [Khopa/dcs_liberation#2697] [Q&A] [answered]
![캡처](https://user-images.githubusercontent.com/107155132/214845542-e6bcc1bd-a16e-46b7-9412-d3e35e20902c.PNG)
I get an error message saying 'cp949'.
Do you know a workaround for this?

**Comment by Starfire13** [comment, marked as answer]
As far as I'm aware, no one else has encountered that error before. What were you doing when it occurred? Does it occur consistently when you do that?

**Reply by MirrorPayne** [reply]
When I press 'BEGIN CAMPAIGN', that message pops up.

**Reply by Starfire13** [reply]
And does this happen every time you hit take off regardless of campaign?

**Reply by MirrorPayne** [reply]
When I first pressed 'BEGIN CAMPAIGN'
If you double-click any of the list in 'Packages', it will appear.

But within dcs the game runs.

Instead, allied planes go into battle unarmed unless a manual loadout is set.

**Reply by Starfire13** [reply]
Try reinstalling Liberation

**Reply by MirrorPayne** [reply]
Even if I delete the folder and unzip it, it is the same.

Do I have to delete it in some other way?

**Reply by Starfire13** [reply]
Deleting the folder is fine.

Make sure you’re also removing the files in C:\Users\username\AppData\Local\dcsliberation\ when you reinstall.

**Reply by MirrorPayne** [reply]
Even if I reinstall, the message pops up, but there is nothing wrong except automatic loadyout, so I'll just do it! Thank you.

**Reply by MirrorPayne** [reply]
underground are automatically armed, but units created in aircraft carriers are not automatically armed and take off with only guns armed.

**Reply by Starfire13** [reply]
Which aircraft? You can always create your own default loadouts btw.

**Reply by MirrorPayne** [reply]
All aircraft are unarmed when departing from an aircraft carrier.

**Reply by MirrorPayne** [reply]
'C:\Users\USER\Desktop\dcs_liberation\resources\customized_payloads'

Putting the file in the path below solved it!

'Games\DCS.openbeta_server\MissionEditor\UnitPayloads'
